src/tlo/methods/healthsystem.py

```diff
@@ -1282,10 +1282,16 @@ def schedule_hsi_event(
         """
         # If there is no specified tclose time then set this to a week after topen.
         # This should be a boolean, not int! Still struggling to get a boolean variable from resource file
-        if self.tclose_overwrite == 1:
-            tclose = topen + pd.to_timedelta(self.tclose_days_offset_overwrite, unit='D')
-        elif tclose is None:
-            tclose = topen + DateOffset(days=7)
+
+        # Clinical time-constraints are embedded in tclose for these modules, do not overwrite their tclose
+        if hsi_event.module == ('CareOfWomenDuringPregnancy' or 'Labour' or 'PostnatalSupervisor' or 'NewbornOutcomes'):
```

Comment:
In this current version, using `or` like that won't give the (I presume) intended behaviour

```suggestion
        if hsi_event.module in ('CareOfWomenDuringPregnancy', 'Labour', 'PostnatalSupervisor' ,'NewbornOutcomes'):
```

Also --- should it be `hsi_event.module.name`???  We're making a comparison to a string for the module's name rather than the module itself.